Database-Features/Error-ODBC-Driver.md

```diff
@@ -0,0 +1,57 @@
+# Error "Can't open lib '/opt/exasol_odbc_driver' : file not found" for odbc driver
+
+## Problem
+
+As Exasol ODBC driver doesn't have / create symlinks itself when a new version of ODBC driver is installed,the user is used to create symlinks to driver libraries to avoid changing ODBC DSN definition ,it's sufficient to update only the symlink.
```

Comment:
```suggestion
As Exasol ODBC driver doesn't have / create symlinks itself when a new version of ODBC driver is installed, the user is used to create symlinks to driver libraries to avoid changing ODBC DSN definition, it's sufficient to update only the symlink.
```